### ISSUE TITLE
🟢 Enable custom tx parameters and remove feature flag

### ISSUE THREAD

**Issue by shivekkhurana** [status-im/status-react#21554]
# Feature Issue

[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a <user|developer|...>, I want to <task> **so that** <goal>. 

### Description

[comment]: # (Describe the feature you would like and include designs and Figma links if available.)
[comment]: # (Describe the use cases the implementation should cover. E.g. New users, existing users, public chat, group chat, etc)
*Summary*: ...

### Acceptance Criteria

[comment]: # (Rules for the future PR to be accepted.)

### Notes

[comment]: # (For the developer who will implement. Include planning poker estimates, guidance, etc.)

**Comment by churik** [comment]
@shivekkhurana given the tight schedule I suppose it can be moved to 2.33, as it is not a blocker

**Comment by churik** [comment]
@alwx are you working on this one?

**Comment by shivekkhurana** [comment]
No, this is meant to be a closing issue when tx params are fully available.

**Comment by vkjr** [comment]
Closed by #22333
